test/e2e/storage/vsphere/config.go

```diff
@@ -98,7 +98,8 @@ func GetVSphereInstances() (map[string]*VSphere, error) {
 
 func getConfig() (*ConfigFile, error) {
 	if confFileLocation == "" {
-		return nil, fmt.Errorf("Env variable 'VSPHERE_CONF_FILE' is not set.")
+		msg := "Env variable 'VSPHERE_CONF_FILE' is not set"
+		return nil, errors.New(msg)
```

Comment:
What is the reason for splitting this into two lines? Is there a specific lint error that recommends this?

Reply:
Yes, golint recommends this

Reply:
fixed